versions.yaml

```diff
@@ -18,17 +18,16 @@
 # CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 versions:
-  - dir: '2/2.32'
+  - dir: '2/2.46.2'
     repo: jenkins2
     tags:
-      - 2.32
-      - 2.32.2
+      - 2.46.2
```

Comment:
The stable branch and latest branch have different versions, it means that `2.46` was never available in stable branch, but in latest it was.

https://jenkins.io/changelog/
https://jenkins.io/changelog-stable/

Maybe should we keep `2`, `2.46.2` and `latest` in this case?